packit/config/aliases.py

```diff
@@ -175,6 +177,153 @@ def get_branches(
     return branches
 
 
+def get_all_ff_branches(
+    dist_git_branches: Union[list, dict, set, None],
+    default: str = DEFAULT_VERSION,
+    default_dg_branch: str = "main",
+    with_aliases: bool = False,
+) -> set[str]:
+    """
+    Returns a list of target branches that can be fast forwarded merging
+    some other branch according to the given dist_git_branches config key.
+
+    The branches should be specified as in the following example
+        {"rawhide": {"open_pull_request_for": ["f40", "f39"]},
+         "epel9": {}
+        }
+
+    dist_git_branches: dist_git_branches config key; can be a list or dict
+    default: the same as get_branches default parameter
+    default_dg_branch: the same as get_branches default_dg_branch parameter
+    with_aliases: the same as get_branches with_aliases parameter
+    """
+    if not isinstance(dist_git_branches, dict):
+        return set()
+
+    ff_branches = set()
+    for open_pull_request_for in dist_git_branches.values():
+        if OPEN_PULL_REQUEST_FOR_KEY in open_pull_request_for:
+            ff_branches.update(
+                get_branches(
+                    *open_pull_request_for[OPEN_PULL_REQUEST_FOR_KEY],
+                    default=default,
+                    default_dg_branch=default_dg_branch,
+                    with_aliases=with_aliases,
+                ),
+            )
+
+    return ff_branches
+
+
+def get_ff_branches_from(
+    dist_git_branches: Union[list, dict, set, None],
+    source_branch: str,
+    default: str = DEFAULT_VERSION,
+    default_dg_branch: str = "main",
+    with_aliases: bool = False,
+) -> set[str]:
+    """
+    Returns a list of target branches that can be fast forwarded merging
+    the specified source_branch
+    The keys can be aliases, expand them into a temporary structure
+    to be sure that source_branch can be found in the first dictionary.
+    In the inner loop of the downstream sync the expanded aliases are used.
+
+    @WARNING If a key alias will be expanded in more than a key, a branch
+    and a PR will be created for every each of them. The user sould choose
+    which one to close or merge.
+
+    The branches should be specified as in the following example
+        {"rawhide": {"open_pull_request_for": ["f40", "f39"]},
+         "epel9": {}
+        }
```

Comment:
On a similar, naming-nitpicky note, I would probably try to make config name and code naming consistent.